### PR TITLE
style text undert the header

### PR DESCRIPTION
**Pull request by Abdalrazaq94**

### CODE REVIEW

File: client/src/pages/Home/WeSupportYou/WeSupportYou.css [hunk 1]
```diff
@@ -15,15 +15,16 @@ Section7
 }
 
 .wesupportyou_header{
-	font-family: 'RobotoSlab-VariableFont_wght';
+	font-family: 'RobotoSlab-Regular';
```
Comment (filkovsp) [inline client/src/pages/Home/WeSupportYou/WeSupportYou.css:13]: hi Meg. 
please add import for RobotoSlab-Regular.ttf into App.css

File: client/src/pages/Home/WeSupportYou/WeSupportYou.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
 import "./WeSupportYou.css";
+import ImgHands from "../../../image/supporting-elderly-woman-2021-09-24-03-07-01-utc.jpg";
```
Comment (filkovsp) [inline client/src/pages/Home/WeSupportYou/WeSupportYou.js:4]: hi Meg, 
please rename file to 
`supporting-elderly-woman.jpg`
we will need to do the same with the rest of the image files

File: client/src/App.css [hunk 3]
```diff
@@ -13,6 +13,22 @@
...
+@font-face {
+    font-family: "NexaTextDemo-Light";
+    src: url("./assets/NexaTextDemo-Light.otf") format("opentype");
+}
+
+@font-face {
+    font-family: "NexaTextDemo-Bold";
+    src: url("./assets/NexaTextDemo-Bold.otf") format("opentype");
+}
+
```
Comment (filkovsp) [inline client/src/App.css:31]: please add import for RobotoSlab-Regular.ttf
and don't forget to place the file into assets folder as well :)

**Review by filkovsp — CHANGES_REQUESTED** [review]

File: client/src/pages/Home/WeSupportYou/WeSupportYou.js (hunk 2 above)
Comment (Megan-Sen) [reply client/src/pages/Home/WeSupportYou/WeSupportYou.js:4]: Thanks for your quick review!!

**Review by filkovsp — APPROVED** [review]

**Pull request merged**